packages/components/components/elvis-stepper/src/react/utils.ts

```diff
@@ -44,32 +44,39 @@ export const numberShouldBeVisible = (
   return false;
 };
 
-export const generateStatusMessage = (currentStep: number, steps: StepStates, errorSteps: number[]) => {
-  const lang = useLanguage();
-
-  const getTranslatedLabels = () => {
-    return {
-      onStep: lang === 'no' ? 'På steg' : 'On step',
-      previousStep: lang === 'no' ? 'Det forrige steget var' : 'The previous step was',
-      steps: lang === 'no' ? 'Steg' : 'Steps',
-      and: lang === 'no' ? 'og' : 'and',
-      wasInvalid: lang === 'no' ? 'var ugyldig' : 'was invalid',
-      statusLabel: (isCompleted: boolean) =>
-        isCompleted
-          ? lang === 'no'
-            ? 'vellykket'
-            : 'successful'
-          : lang === 'no'
-            ? 'ikke fullført'
-            : 'not completed',
-    };
-  };
+export const generateStatusMessage = (
+  currentStep: number,
+  steps: StepStates,
+  errorSteps: number[],
+  lang: LanguageCode,
+) => {
+  const labels = (() => {
+    return lang === 'no'
+      ? {
+          onStep: 'På steg',
+          previousStep: 'Det forrige steget var',
+          steps: 'Steg',
+          and: 'og',
+          wasInvalid: 'var ugyldig',
+          successful: 'vellykket',
+          notCompleted: 'ikke fullført',
+        }
+      : {
+          onStep: 'On step',
+          previousStep: 'The previous step was',
+          steps: 'Steps',
+          and: 'and',
+          wasInvalid: 'was invalid',
+          successful: 'successful',
+          notCompleted: 'not completed',
+        };
+  })();
```

Comment:
There shouldn't be any need for this to be an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE), so just convert it to a regular object maybe?